packages/client/modules/userDashboard/components/OrgAuthentication/OrgAuthenticationMetadata.tsx

```diff
@@ -40,32 +43,50 @@ const ButtonSection = styled('div')({
 
 interface Props {
   disabled: boolean
+  orgId: string
 }
 
 const OrgAuthenticationMetadata = (props: Props) => {
   const [metadata, setMetadata] = useState('')
-  const {disabled} = props
+  const {disabled, orgId} = props
+  const {onError, onCompleted, submitMutation, submitting} = useMutationProps()
```

Comment:
-1 we'll need to also grab `error` and show that in an error message. For example, if the metadata is invalid, we'll watch to validate it on the server & then show a message that it is invalid here